app/controllers/users_controller.rb

```diff
@@ -1,18 +1,23 @@
 class UsersController < ApplicationController
+  before_action :authenticate_user!, only: [:show]
   before_action :find_user, only: [:show]
 
   def index
     @users = User.includes(:posts).all
   end
 
```

Comment:
**[OPTIONAL BUT HIGHLY RECOMMENDED]**

- [ ] It would be nice if you make your own logo of your name and put that in the readme file at the top to present yourself as a brand in the job market or tech industry to catch the recruiters' eyes 👍